src/ViewInstance.js

```diff
@@ -139,6 +141,13 @@ class ViewInstance extends React.Component {
         query: '(module==SETTINGS and configName==TLR)',
       },
     },
+    orders: {
```

Comment:
Is it required? Maybe, make `ky.get` request for order number validation in `NewOrderModalContainer` component instead?